tests/unit/test_dbapi.py

```diff
@@ -256,3 +256,15 @@ def test_tags_are_set_when_specified(mock_client):
     # THEN
     _, passed_client_tags = mock_client.ClientSession.call_args
     assert passed_client_tags["client_tags"] == client_tags
+
+
+@patch("trino.dbapi.trino.client")
+def test_role_is_set_when_specified(mock_client):
+    # WHEN
```

Comment:
seems redundant comment

Reply:
Can we inspect current role for user from the server using a query?

If so I'd prefer if the test used it instead of just asserting arguments are passed correctly.

Reply:
We could test it with Hive but that means we would have to change our docker setup to use docker compose.

```
❯ trino --user admin
trino> show current roles in hive;
  Role  
--------
 public 
(1 row)

Query 20220923_160052_00000_kq3fp, FINISHED, 1 node
Splits: 1 total, 1 done (100,00%)
1,51 [0 rows, 0B] [0 rows/s, 0B/s]

trino> show current roles;
 Role 
------
(0 rows)

Query 20220923_160107_00001_kq3fp, FINISHED, 1 node
Splits: 1 total, 1 done (100,00%)
0,05 [0 rows, 0B] [0 rows/s, 0B/s]

trino> create role public;
Query 20220923_160134_00002_kq3fp failed: System roles are not enabled

```

Reply:
Can you create an issue about integration tests for roles and we can discuss how to add them as follow up (since once we have them we'd probably want to use it for other tests too).

Reply:
Done